src/Sarif.Multitool/TransformOptions.cs

```diff
@@ -10,10 +10,10 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     internal class TransformOptions : SingleFileOptionsBase
     {
         [Option(
-            't',
+            'v',
             "target-version",
             HelpText = "The SARIF version to which the input file will be transformed.",
-            Default = 2)]
-        public int Version { get; internal set; }
+            Default = SarifVersion.Current)]
+        public SarifVersion Version { get; set; }
```

Comment:
>Version [](start = 28, length = 7)

There are two versions in play, the source and the target. And you helpfully named the command line option `--target-version`. So IMO the property should also be named `TargetVersion` (and then IMO the shortcut option should still be `-t`, to match).

Reply:
-v is easy to remember as the 'version'. -t is not. another idea here is to forbid a short version altogether. or we could prefer -v and --version

---
In reply to: [229509857](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229509857) [](ancestors = 229509857)

Reply:
I'd accept `-v` as the short option name but I feel pretty strongly from a code clarity standpoint that the property should be named `TargetVersion`.

---
In reply to: [229513921](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229513921) [](ancestors = 229513921,229509857)

Reply:
I strongly agree with you. to be clear, you're ok with 'v' and --target-version and TargetVersion?

---
In reply to: [229514757](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229514757) [](ancestors = 229514757,229513921,229509857)

Reply:
Yes.

---
In reply to: [229515780](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229515780) [](ancestors = 229515780,229514757,229513921,229509857)